### PR TITLE
Introduce polling for live update of messages and notifications

### PR DESCRIPTION
**Pull request by sonalkr132**
- [x] update to rails 4.2.5.1
- [x] live messages
- [x] live notification
- [x] tests

### CODE REVIEW

**Comment by sonalkr132** [comment]
Guide followed for update to rails 4.2.5.1: http://edgeguides.rubyonrails.org/upgrading_ruby_on_rails.html#upgrading-from-rails-4-1-to-rails-4-2

`web-console` and `responders` was not added cause we don't really need it. We might consider introducing `spring`, it has been included by default since rails 4.1, however rails 5 seems to have issues it for now.

One of the crucial updates is update of devise to 3.5.5 due to security vulnerable. You can read about it [here](http://blog.plataformatec.com.br/2016/01/improve-remember-me-cookie-expiration-in-devise/).

**Comment by sonalkr132** [comment]
Phew! Finally travis is green. Drop in test coverage is due to update of `simplecov`. And they are right, some of our files have 0% which weren't included in test coverage previously, they are included now.
However, I won't be working on test coverage in this PR.

File: app/assets/javascripts/comments.js.coffee [hunk 1]
```diff
@@ -0,0 +1,17 @@
+@CommentPoller =
+	poll: ->
+		setTimeout @request, 30000
```
Comment (Alwahsh) [inline app/assets/javascripts/comments.js.coffee]: How about putting that number in some variable for configuration? I don't think it should be hard coded.

Comment (sonalkr132) [inline app/assets/javascripts/comments.js.coffee]: Thanks @Alwahsh . I have made the required change.

**Pull request merged**